rasa/core/actions/loops.py

```diff
@@ -11,7 +11,10 @@
     from rasa.core.trackers import DialogueStateTracker
 
 
-class LoopAction(Action, ABC):
+class LoopAction(Action, ABC):  # pytype: disable=base-class-error
+    def name(self) -> Text:
+        raise NotImplementedError
```

Comment:
is it really required to redefine `name`? It's actually the same implementation as in `Action`?

Reply:
@wochinge you implemented it 😅

Reply:
I have no idea

Reply:
@tabergma added it fix deepsource warnings I guess 🕵️

Reply:
Yes, otherwise we get attribute error as the method `self.name()` is not defined

Reply:
wait, but this is always subclassed and then `self.name()` has to be re-implemented. When do you get the `AttributeError`?

Reply:
It was the type check that was failing. Pytype does not understand the subclassing of this class, thus I had to add the "ignnore base-class-errror" and I guess due to that it also did not correctly recognized that the method `self.name()` came from the super class. So we can either add this abstract method or add a "ignore attribute-error" around the lines where `self.name()` is used.

Reply:
Omg. What's up with `pytype`. It passed on this before - I mean it lives on `master` for a while now. I'd prefer to add the `ignore` instead of adding code just for the sake of a tool.